apps/condo/migrations/20230419122709-0261_ticket_feedbackadditionaloptions_and_more.js

```diff
@@ -0,0 +1,253 @@
+BEGIN
+    if option = 'Сделали на совесть' or option = 'Made in good faith' then return 'highQuality';
+    elseif option = 'Быстро сделали' or option = 'Quickly done' then return 'quickly';
+    elseif option = 'Сделали некачественно' or option = 'It was done poorly' then return 'lowQuality';
+    elseif option = 'Долго делали' or option = 'It took a long time' then return 'slowly';
+    else return '';
+    end if;
+end
+$getFeedbackOptionKeyFromReviewCommentOption$ LANGUAGE plpgsql;
+
+
+CREATE OR REPLACE FUNCTION convertReviewCommentToFeedbackAdditionalOptions(value text)
+    returns jsonb
+as
+$convertReviewCommentToFeedbackAdditionalOptions$
+declare
+    key1 text;
+    key2 text;
+BEGIN
+    key1 := getFeedbackOptionKeyFromReviewCommentOption(split_part(value, ';', 1));
+    key2 := getFeedbackOptionKeyFromReviewCommentOption(split_part(value, ';', 2));
+
+    if key1 = key2 and key1 != '' then return format('["%s"]', key1)::jsonb;
+    elseif key1 = key2 and key1 = '' then return null::jsonb;
+    elseif key1 != '' and key2 != '' then return format('["%s","%s"]', key1, key2)::jsonb;
+    elseif key1 = '' and key2 != '' then return format('["%s"]', key2)::jsonb;
+    elseif key1 != '' and key2 = '' then return format('["%s"]', key1)::jsonb;
+    else return null::jsonb;
+    end if;
+end
+$convertReviewCommentToFeedbackAdditionalOptions$ LANGUAGE plpgsql;
+
+--
+-- [CUSTOM] Set feedbackValue from reviewValue for Ticket
+--
+
+UPDATE public."Ticket"
+    SET "feedbackValue" = "reviewValue"
+    WHERE "reviewValue" IS NOT NULL;
+
+--
+-- [CUSTOM] Set feedbackAdditionalOptions from reviewComment for Ticket
+--
+
+UPDATE public."Ticket"
+    SET "feedbackAdditionalOptions" = convertReviewCommentToFeedbackAdditionalOptions("reviewComment")
+    WHERE "reviewComment" IS NOT NULL AND "reviewComment" != '';
+
+--
+-- [CUSTOM] Set feedbackValueFrom from reviewValueFrom for TicketChange
+--
+
+UPDATE public."TicketChange"
+    SET "feedbackValueFrom" = "reviewValueFrom"
+    WHERE "reviewValueFrom" IS NOT NULL;
+
+--
+-- [CUSTOM] Set feedbackValueTo from reviewValueTo for TicketChange
+--
+
+UPDATE public."TicketChange"
+    SET "feedbackValueTo" = "reviewValueTo"
+    WHERE "reviewValueTo" IS NOT NULL;
+
+--
+-- [CUSTOM] Set feedbackAdditionalOptionsFrom from reviewCommentFrom  for TicketChange
+--
+
+UPDATE public."TicketChange"
+    SET "feedbackAdditionalOptionsFrom" = convertReviewCommentToFeedbackAdditionalOptions("reviewCommentFrom")
+    WHERE "reviewCommentFrom" IS NOT NULL AND "reviewCommentFrom" != '';
+
+--
+-- [CUSTOM] Set feedbackAdditionalOptionsTo from reviewCommentTo  for TicketChange
+--
+
+UPDATE public."TicketChange"
+    SET "feedbackAdditionalOptionsTo" = convertReviewCommentToFeedbackAdditionalOptions("reviewCommentTo")
+    WHERE "reviewCommentTo" IS NOT NULL AND "reviewCommentTo" != '';
+
+--
+-- [CUSTOM] Set feedbackValue from reviewValue for TicketFilterTemplate
+--
+
+UPDATE public."TicketFilterTemplate"
```

Comment:
You also need to check the front end side. If you use the old front end version. If should not get any runtime errors